src/geckomat/change_model/getKcatAcrossIsoenzymes.m

```diff
@@ -0,0 +1,38 @@
+function model = getKcatAcrossIsoenzymes(model)
+% getKcatAcrossIsoenzymes
```

Comment:
I guess this is only relevant for full ec models? May be good to write in the text and check in the function.

Reply:
Will you add a test case for this function? It should be relatively easy - you could copy parts of tc0011 and just zero the kcat for reaction R2_EXP_2 or something like that. And then make sure it gets filled in from R2_EXP_1. Make sure it works also for REV.

Reply:
Or would you want something similar for light models? Then the smallest isoenzyme will always be selected, while if that isoenzyme would not have had a kcat value it would never be active? (I assume that light ignores isoenzymes with 0 kcat value?)

Reply:
Yeah, they are ignored, but then it would be the mean of the other possibilities, and it would not be the smallest, and not be used :). I don't think we need one here - it is only if you constrain individual enzymes that this makes sense unless I misunderstood something?

Reply:
Right, indeed, I'll then just let it throw an error if applied to geckoLight model.